### ISSUE TITLE
{USERNAME} in kits is not displayed completely

### ISSUE THREAD

**Issue by danechek** [EssentialsX/Essentials#2953]
The name displayed by the {USERNAME} placeholder in kits is displayed without the last character.
Example, let's say the name of the player **Gamer**: 
- apple 1 lore: owner {USERNAME}
Displayed text in lore of apple:  owner **Game**

17.01 11:16:20 [Server] INFO Server version: 1.15.1-R0.1-SNAPSHOT git-Paper-50 (MC: 1.15.1)
17.01 11:16:20 [Server] INFO EssentialsX version: 2.17.1.50
17.01 11:16:20 [Server] INFO LuckPerms version: 5.0.39
17.01 11:16:20 [Server] INFO PlaceholderAPI version: 2.10.4
17.01 11:16:20 [Server] INFO Vault version: 1.7.2-b107
17.01 11:16:20 [Server] INFO EssentialsXChat version: 2.17.1.50
17.01 11:16:20 [Server] INFO EssentialsXSpawn version: 2.17.1.50

**Comment by triagonal** [comment]
I'm unable to reproduce this on the following versions:
```
[08:29:36 INFO]: CONSOLE issued server command: /ess ver
[08:29:36 INFO]: Server version: 1.15.1-R0.1-SNAPSHOT git-Paper-42 (MC: 1.15.1)
[08:29:36 INFO]: EssentialsX version: 2.17.1.52
[08:29:36 INFO]: LuckPerms version: 5.0.52
[08:29:36 INFO]: Vault version: 1.7.2-b107
```
The following kit displays the player's username correctly (the player's username is ExplodingGravy):
```
loretest:
  delay: -1
  items:
    - pumpkinpie 1 lore:This_belongs_to_{USERNAME}
```
![image](https://user-images.githubusercontent.com/10545540/72844866-85ee6700-3cf1-11ea-8492-a378ffed1b4b.png)

This _might_ be caused by how you've defined your kit item. To properly insert a space in a lore string, you should use an underscore ( _ ) instead of a regular space.

**Comment by danechek** [comment]
I understood what the problem is. I use "|" for a new line, but if you write it immediately after {USERNAME}, you get this bug. Have to write {USERNAME}_|

**Comment by danechek** [comment]
This is a new bug. Previously, this was not, it turns out this is happening not only with {USERNAME}. "|" wraps the text on a new line, but removes the last character on the last.

**Comment by pop4959** [comment]
Closed in favour of #2962. Please follow that issue for further updates on this.
